README.md

```diff
@@ -1 +1,111 @@
-# covid-tracker
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Covid Tracker
+
+## About
+
+This is a covid tracker app where you can find latest information about COVID in Brazil, total number of cases, today new confirmed cases and deaths and more, you can also search by state and check specific information about any state of Brazil.
+
+Our development flow included: performing git flow practices & using components with routers, life cycle methods,handling events and state management by Redux store,actions, reducers in [duck Patern](https://github.com/erikras/ducks-modular-redux) with unit and component tests.
+
+## Built With
+
+- Javascript
+- React & Redux
+- Redux Middleware (Thunk, Logger)
+- Bootstrap
+- Jest for testing
+
+## Live Demo
+
+[Live Demo Link](https://blissful-northcutt-e13aab.netlify.app/)
+
+## Presentation video
+
+[Watch video](https://www.loom.com/share/bff86c2832e348889c00439235d390d7)
+
+## Getting Started
+
+To get a local copy up and run the app; follow below steps.
+
+### Prerequisites
+
+If you are not familiar to use terminal, please check this [article](https://www.theodinproject.com/courses/web-development-101/lessons/command-line-basics-web-development-101)
+
+### Setup
+
+Open your terminal and run the following commands on your terminal.
+
+Make sure 'Javascript' is not blocked in your browser.
+
+### Install
+
+To get a local copy of the repository, run the following commands on your terminal.
+
+```
+cd <your_desktop>
+```
+
+```
+git clone git@github.com:yigitm/react-group-project.git
+```
+
+```
+cd react-group-project
+```
```

Comment:
- [ ] You have some typo errors in your setup link. This is the correct link: **`git@github.com:jonathastavares/covid-tracker.git`**. Also change line 54 accordingly.